### PR TITLE
fix: issues with the demo in ie11 and firefox

### PR DESCRIPTION
**Pull request by brandonocasey**
## Description
* Adds back in url type specifier
* Fixes issues and styling in firefox
* fixes dispatch event in ie 11

### CODE REVIEW

**Comment by gkatsev** [comment]
This isn't working for me locally or on netlify

File: scripts/index.js [hunk 1]
```diff
@@ -19,6 +19,19 @@
     console.warn('unhandled input type ' + el.type);
   };
 
+  var newEvent = function(name) {
+    var event
+
+    if (typeof window.Event === 'function') {
+      event = new window.Event('submit');
```
Comment (gkatsev) [inline scripts/index.js]: I think this should be `name` not `'submit'`

Comment (brandonocasey) [reply scripts/index.js]: yep

**Comment by gkatsev** [comment]
Tests didn't pass but no actual code that is tested was changed.

**Pull request merged**